src/api/vscode.ts

```diff
@@ -0,0 +1,3 @@
+import * as vscode from 'vscode';
+
+export const registerDocumentSymbolProvider = vscode.languages.registerDocumentSymbolProvider;
```

Comment:
How is this different than using `vscode.languages.registerDocumentSymbolProvider` directly from Joyride?

Reply:
The symbols get grouped differently. Each extension gets their own top level group in the outline view.

Reply:
So `vscode.languages.registerDocumentSymbolProvider`, when used by some consumer outside of Calva, will register symbols under Calva? I guess in that case it's a closure over some Calva details or something..